src/importedentity.cpp

```diff
@@ -43,11 +45,13 @@ ImportedEntity::~ImportedEntity()
 
 bool ImportedEntity::isImport() const
 {
+    // return !mPimpl->mImportSource.expired();
```

Comment:
random question, but these changes are due to the new use of `shared_from_this` for the import sources...or something, right?

Reply:
Yes.  We now have a strong connection between `ImportedEntity` items and their `ImportSource`, and a weak connection between the `Model` and the `ImportSource`s it contains.

Reply:
I've removed the comment in 6e455f8c